provision/initramfs/init

```diff
@@ -351,7 +351,7 @@ if [ -n "$WWPOSTSHELL" -a "$WWPOSTSHELL" != "0" ]; then
     setsid /bin/cttyhack /bin/sh
 fi
 
-if [ -x "$NEWROOT/sbin/init" ]; then
+if [ -x "$NEWROOT/sbin/init" ] || [ -h "$NEWROOT/sbin/init" -a -x "$NEWROOT`readlink $NEWROOT/sbin/init`" ]; then
```

Comment:
Well... the only think I can think of is, by using readlink, you're verifying that the target `$NEWROTO/sbin/init` symlink points to is executable. Speaking of...

@macdems On the list you mentioned changing the symlink to a relative path instead of absolute. I don't see that happening in your `post_chroot()` (include-ubuntu) function. Was that "fix" going to be added in, or was it just mentioned as something to be aware of?